Source/init.cpp

```diff
@@ -93,8 +94,21 @@ std::vector<std::string> GetMPQSearchPaths()
 		paths.pop_back();
 
 #if defined(__unix__) && !defined(__ANDROID__)
-	paths.emplace_back("/usr/share/diasurgical/devilutionx/");
-	paths.emplace_back("/usr/local/share/diasurgical/devilutionx/");
+	// `XDG_DATA_HOME` is usually the root path of `paths::PrefPath()`, so we only
+	// add `XDG_DATA_DIRS`.
+	const char *xdgDataDirs = std::getenv("XDG_DATA_DIRS");
+	if (xdgDataDirs != nullptr) {
+		for (const string_view path : SplitByChar(xdgDataDirs, ':')) {
+			std::string fullPath(path);
+			if (!path.empty() && path.back() != '/')
+				fullPath += '/';
+			fullPath.append("diasurgical/devilutionx/");
+			paths.push_back(std::move(fullPath));
+		}
+	} else {
+		paths.emplace_back("/usr/share/diasurgical/devilutionx/");
+		paths.emplace_back("/usr/local/share/diasurgical/devilutionx/");
+	}

```

Comment:
According to the spec, the default value for `XDG_DATA_DIRS` is `/usr/local/share:/usr/share`, i.e. the same as these paths but in reverse order. So I guess you just assume `/usr/local/share:/usr/share` if `XDG_DATA_DIRS` is empty or unset.

Reply:
Good point, I've fixed the order for the default to be the same as what's in `XDG_DATA_DIRS` by default.